services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -464,6 +463,15 @@ private UserConsentModel createConsent(ClientModel client, ConsentRepresentation
         return consent;
     }
 
+    @Path("/applications")
+    public ApplicationResource applications() {
+        checkAccountApiEnabled();
+//        auth.requireOneOf(AccountRoles.MANAGE_ACCOUNT, AccountRoles.VIEW_PROFILE);
```

Comment:
@douglaspalmer is this temporary?

Reply:
Btw I commented out the following snippet:

```
@Path("/applications")
    public ApplicationResource applications() {
        checkAccountApiEnabled();
//        auth.requireOneOf(AccountRoles.MANAGE_ACCOUNT, AccountRoles.VIEW_PROFILE);
        return new ApplicationResource(session, user, auth, request);
    }
```

And I could confirm that the endpoint responding is https://github.com/keycloak/keycloak/blob/c6c5a427537b868640d31310505f7a6411bd861c/services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java#L251